openapi_python_client/__init__.py

```diff
@@ -157,40 +137,17 @@ def _run_command(self, cmd: str) -> None:
         cmd_name = cmd.split(" ")[0]
         command_exists = shutil.which(cmd_name)
         if not command_exists:
-            self.errors.append(
-                GeneratorError(
-                    level=ErrorLevel.WARNING, header="Skipping Integration", detail=f"{cmd_name} is not in PATH"
-                )
-            )
+            log.warning("Skipping integration: %s is not in PATH", cmd_name)
             return
+        cwd = self.package_dir if self.meta == MetaType.NONE else self.project_dir
         try:
-            cwd = self.package_dir if self.meta == MetaType.NONE else self.project_dir
             subprocess.run(cmd, cwd=cwd, shell=True, stdout=subprocess.PIPE, stderr=subprocess.PIPE, check=True)
```

Comment:
why would we call any process?

Reply:
I think it's used to run post hooks, formatters/lint. It's from the original generator:
```
post_hooks: List[str] = [
        "autoflake -i -r --remove-all-unused-imports --remove-unused-variables .",
        "isort --float-to-top .",
        "black .",
    ]
```
Not really neccessary to run these in subprocess, we can call them in python